bank_statement_parse_nl_rabo_mt940/__openerp__.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    This module copyright (C) 2014-2015 Therp BV <http://therp.nl>.
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    "name": "MT940 import for dutch Rabobank",
+    "version": "1.1",
+    'author': 'Odoo Community Association (OCA), Therp BV',
+    'website': 'https://github.com/OCA/bank-statement-import',
+    "category": "Account Banking",
+    "depends": [
+        'bank_statement_parse_mt940'
+    ],
+    "data": [],
```

Comment:
Thank you for your vigilant review! With regards to this particular comment, I was wondering what the rationale is for removing empty entries from the manifest file. A lot of people that I know use some sort of scaffolding for the Odoo module and manifest file structure, which leads to these entries. I don't remember any review in which this was a problem. Can we be more lenient on this point?